Smarty/templates/BrowserVariables.tpl

```diff
@@ -12,6 +12,7 @@
 var gVTModule = "{$GVTMODULE}";
 var gVTModuleLabel = "{$GVTMODULELABEL}";
 var gVTTheme  = "{$THEME}";
+var gVTModuleLabel = "{$GVTMODULELABEL}";
```

Comment:
next time make this commit and the change in include/utils/utils.php in a separate commit. we try to make small commits which are semantically related, so this one would have been

`feat(BrowserVariables) add module label`

you add that feature then you do another commit with the menu change that uses this feature

https://discussions.corebos.org/documentation/doku.php?id=en:devel:commitguidelines